.gitignore

```diff
@@ -1,16 +1,106 @@
-# Deployment files
-*.egg-info
-dist
-
-# Environment files
-site-packages
-build
-env
-.cache
-*.so
+# source: https://github.com/github/gitignore/blob/master/Python.gitignore
```

Comment:
I find this too verbose/much.
Most of it makes sense in a user's global gitignore already/only.